src/coreclr/tools/aot/crossgen2/crossgen2.csproj

```diff
@@ -46,4 +46,18 @@
   <Import Project="$(CoreCLRBuildIntegrationDir)Microsoft.DotNet.ILCompiler.targets"
           Condition="'$(NativeAotSupported)' == 'true' and '$(RunningPublish)' == 'true'" />
 
+  <Target Name="RewriteRuntimePackDir"
+          Condition="'$(RunningPublish)' == 'true'"
+          DependsOnTargets="ResolveRuntimeFilesFromLocalBuild"
+          BeforeTargets="ResolveRuntimePackAssets">
+      <ItemGroup>
+        <!-- Remove AspNetCore runtime pack since we don't build it locally -->
```

Comment:
Is removing the AspNetCore runtime pack actually necessary? I never hit issue with it being available as long as no projects use it.

Reply:
ping @agocke for that question

Reply:
@agocke did you see that question? :)